### PR TITLE
Check unique url match

### PR DESCRIPTION
**Pull request by dotmancando**
add function to check:
```When the testharness uses an individual thing's DataURL the resulting slice of things MUST contain a Thing with that same individual DataURL. It MAY also contain other things if that single fetch happens to also retrieve them.```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,49 @@ Light weight tool for fetcher develpers to run real tests locally without Pomelo
...
+Then for each URL to fetch, get one dataURL from first thing returned to access
+```
+fetchAllInterval := time.Duration(3) * time.Second // interval between each Fetch
+urlsToFetch := 10                                  // total of urls to Fetch
+harness.RunAll(context.Background(), fetchAllInterval, urlsToFetch)
+```
+
+
+### RunFetch
+RunFetch only fetch the spedified URLs
```
Comment (smulube) [inline README.md]: spedified - specified

File: README.md [hunk 2]
```diff
@@ -20,11 +20,49 @@ Light weight tool for fetcher develpers to run real tests locally without Pomelo
...
 * change the `template` in the main.go to the package name that you are testing
     * one in import `"bitbucket.com/thingful/template"`
     * another one in main function `harness, err := testharness.Register(template.NewFetcher)`
 * run `glide up`
 * run by `make harness`
 
+## How to use
+The code that runs TestHarness is at `com/harness/main.go`. This code is called by `make harness` command
+
+There are 3 main features, these funtions can be used separately.
```
Comment (smulube) [inline README.md]: funtions -> functions

**Review by smulube — CHANGES_REQUESTED** [review]
couple of v.minor spelling corrections, but other than that I think this looks good to merge I think.

**Comment by dotmancando** [comment]
fixed typo from Sam's comment

**Review by smulube — APPROVED** [review]

**Pull request merged**